prime-router/metadata/fhir_mapping/hl7/resource/MessageHeader.yml

```diff
@@ -14,7 +14,9 @@ eventCoding:
 source:
   valueOf: secondary/Source
   expressionType: resource
+  specs: MSH.3 | MSH.24
```

Comment:
the `|` syntax will short circuit on the first available input. So if MSH.3 is present then it will be used for `source`. Otherwise, MSH.24 will be used.

Reply:
The mapping inventory recommened defaulting to `MSH.24`. But there was a comment in the mapping saying 
`We believe that if MSH-24 is vaued, although it is rarely valued, it contains better information for the MessageHeader.source.endpoint.  However, one may either still want to use MSH-3 in total or a combination of content from MSH-3 and MSH-24.  That is a local implementation decision to adjust the mapping accordingly.`

Based on public health knowledge `MSH.3` is of higher usage and should be preferred over `MSH.24`

Reply:
Can we leave a comment indicating for posterity capturing the decision?